compiler/optimizer/LocalReordering.cpp

```diff
@@ -357,6 +357,7 @@ void TR_LocalReordering::moveStoresEarlierIfRhsAnchored(TR::Block *block, TR::Tr
        !node->getOpCode().isTreeTop() &&
        !node->getOpCode().isCall() &&
        (node->getOpCodeValue() != TR::New) &&
+       (node->getOpCodeValue() != TR::newvalue) &&
```

Comment:
Was this spot chosen in particular for some reason ? i.e. was the rest of the optimizer checked for places where new* opcodes are dealt with and only this one mattered for this change ? Maybe the point is that this ought to have been changed along with N other places using new* opcodes that were changed properly and this commit is just fixing that bug.